cuppa/src/main/python/pycuppa/cuppa/visualization/plot_predictions.R

```diff
@@ -515,7 +524,7 @@ plot_feat_contrib <- function(VIS_DATA){
       feat_value_label[feat_value==0] <- "0"
       feat_value_label[feat_value==1] <- "1"
 
-      paste0(affixes$feat_basename, " = ", feat_value_label)
+      paste0(ifelse(grepl("\\.", affixes$feat_basename), paste(gsub("\\.", " ", affixes$feat_basename), " (", feat_value_label, ")", sep = ""), ifelse((affixes$feat_basename == "is male") | (affixes$feat_basename == "whole genome duplication"), paste(affixes$feat_basename, "=", feat_value_label, sep = " "), paste(affixes$feat_basename, "=", feat_value_label, sep = " "))))
```

Comment:
This could be split into several steps to make it easier to read.

Also `(affixes$feat_basename == "is male") | (affixes$feat_basename == "whole genome duplication")` can be replaced with `affixes$feat_basename %in% c("is male", "whole genome duplication")`